src/basic/user-util.c

```diff
@@ -668,6 +668,15 @@ bool valid_home(const char *p) {
         return true;
```

Comment:
maybe add a very quick comment to valid_home() that the same code is used for valid_shell(), so that everybody who modifies it keeps that in mind...